tests/base.py

```diff
@@ -60,6 +61,10 @@ def setUp(self):
             location_two.save()
             location_three = Location(name='Lagos', abbreviation='LOS')
             location_three.save()
+            structure = OfficeStructure(name="OfficeA")
```

Comment:
I think it will be better if we named this node or level instead of `structure`. Otherwise good work

Reply:
Thanks, @nakatuddesuzan, this has been implemented